src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -1072,8 +1075,16 @@ void mongoSubCacheStatisticsGet(int* refreshes, int* inserts, int* removes, int*
       char          msg[256];
       unsigned int  bytesLeft = listSize - strlen(list);
 
-      snprintf(msg, sizeof(msg), "%s", cSubP->subscriptionId);
-
+#if 0
```

Comment:
It seems this PR doesn't solve the https://github.com/telefonicaid/fiware-orion/issues/1406, so this will remain.

NTC